### PR TITLE
Run CCI in a subprocess

### PR DESCRIPTION
**Pull request by wlonk**
Run org setup flows in a subprocess that doesn't have access to most of the environment.

I think this is ready.

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -8,4 +8,4 @@ exclude =
     coverage,
     node,
     node_modules
-max-line-length=88
+max-line-length=100
```
Comment (wlonk) [inline .flake8]: Is this a change we want to make? We should make a matching change where we call `black` then. I'll push that.

**Comment by wlonk** [comment]
@davisagli I made one small change to match line lengths with what you had configured. If you give it another 👍 I'll merge it.

File: metecho/api/sf_run_flow.py [hunk 2]
```diff
@@ -279,12 +283,45 @@ def create_org(
...
 
-def run_flow(*, cci, org_config, flow_name, project_path):
+def run_flow(*, cci, org_config, flow_name, project_path, user):
     """Run a flow on a scratch org"""
-    # Run flow (takes care of getting a new access token)
-    flow = cci.get_flow(flow_name)
-    with cd(project_path):
-        flow.run(org_config)
+    # Run flow in a subprocess so we can control the environment
+    gh_token = user.socialaccount_set.get(provider="github").socialtoken_set.get().token
```
[the remark this answers is not in the document]
Comment (davisagli) [reply metecho/api/sf_run_flow.py]: Probably, but can we merge this as is and come back to that?

[the remark this answers is not in the document]
Comment (wlonk) [reply metecho/api/sf_run_flow.py]: Given that we have `User.sf_token`, I'd love to also have `User.gh_token`.

**Pull request merged**